Posts/2021/02/Get-Services, PowerShell 7 VS Windows PowerShell 5.1

```diff
@@ -0,0 +1,150 @@
+---
+post_title: Get-Services, PowerShell 7 VS Windows PowerShell 5.1
+username: farisnt@gmail.com
+Catagories: PowerShell, Windows Services
+Summary: Show how to get services information using both Windows PowerShell 5.1 and PowerShell 7
+---
+
+# Get-Services, PowerShell 7 VS Windows PowerShell 5.1
+
+**Q**: How can I get the Username, and StartType for a Windows Service?
+
+**A**: Quick answer is PowerShell 7.
```

Comment:
The answer is to use PowerShell 7 (or use WMI()

Reply:
Yes, surely also via WMI, but the thing I want to show is how easy it's via PowerShell 7 as the user won't need to use any WMI query.
But it's totally fine, what do you think if we make it like
The quick answer is PowerShell 7, or a longer answer is WMI. 
I am OK with whatever you see better :)